src/label.js

```diff
@@ -29,9 +29,10 @@ function boundingRects(model) {
 	};
 }
 
-function getScaleOrigin(el) {
-	var horizontal = el._model.horizontal;
-	var scale = el._scale || (horizontal && el._xScale) || el._yScale;
+function getScaleOrigin(el, context) {
+	var horizontal = el.horizontal;
+	var meta = context.chart.getDatasetMeta(context.datasetIndex);
+	var scale = meta.vScale;
```

Comment:
It stands for `value`. I do agreed it could be more explicit. 
Quick search: 10 times `.iScale`, 14 times `.vScale`
So renaming those to `indexScale` and `valueScale` would cost 96 bytes. probably worth it.

Reply:
@etimberg any thoughts about the naming here?

Reply:
I agree, `indexScale` and `valueScale` worth the extra cost compared to readibility and maintainability.